framework/Sources/VGSFramework/UIElements/Text Field/Mask/MaskedTextField.swift

```diff
@@ -53,12 +53,15 @@ open class MaskedTextField: UITextField {
         set {
             super.text = newValue
             self.formatText()
+            self.onEditingChange?()
```

Comment:
Noooo. Named incorrect. This is the JS style of naming. Say no for `on...` or `click...` )) 
I guess will be good `self.editingChange?()`

Reply:
And need to add `[weak self] in` Or maybe I'm wrong?